test/functional/ui/cmdline_spec.lua

```diff
@@ -23,86 +28,59 @@ local function test_cmdline(linegrid)
 
   it('works', function()
     feed(':')
-    screen:expect {
-      grid = [[
-      ^                         |
-      {1:~                        }|*3
-                               |
-    ]],
+    screen:expect({
+      grid = empty_grid,
       cmdline = { {
         firstc = ':',
         content = { { '' } },
         pos = 0,
       } },
-    }
+    })
 
     feed('sign')
-    screen:expect {
-      grid = [[
-      ^                         |
-      {1:~                        }|*3
-                               |
-    ]],
+    screen:expect({
```

Comment:
Should still use `expect { }` syntax (the function takes several named args, not a single table arg)

Reply:
Do you mean to omit the surrounding `()`? How does that better indicate the function takes several named args?

I added them because we seem to prefer surrounding parentheses in general (and may want to enforce it at some point)? Previous discussion: https://github.com/neovim/neovim/pull/31441#discussion_r1869476420.

Reply:
the general style _is_ to use `expect { }` and has always been. as stated in the linked thread this is what is directly enforced by `screen:snapshot_util()` already. 

Hacking things away from this output for the sake to be different in individual PRs does not make things more "consistent" or "general"

Reply:
> the general style is to use expect { } and has always been. as stated in the linked thread this is what is directly enforced by screen:snapshot_util() already.

The thing is that that already is no longer the case, as stated in the linked thread... But if you feel strongly about this some previous PRs should be reverted instead.

Reply:
for reference, I wrote this as a summary the last time we did a _consistent_ major refactor (the enabling of stylua in the first place): https://github.com/JohnnyMorganz/StyLua/issues/829 . So, yes the part of "correcting" the line breaks in snapshot_util() to match what stylua does for now, did make sense. However, when stylua was enabled #26398 it already supported our existing and fully consistent style of brackets, so there was no need to change that. 

Unless I missed some other discussion, I just don't see how pointless churn in unrelated PRs back and forth helps towards consistency or readability, thus I just simply suggested to avoid that here and in further PR:s:p

Reply:
> it already supported our existing and fully consistent style of brackets, so there was no need to change that.

As I mentioned before, it is _not_ fully consistent in `/test`, and our _more_ scrutinized `/runtime/lua` consistently uses call parentheses (as far as I could tell last time). I changed it to match that.